mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/UtbetalingService.kt

```diff
@@ -81,6 +92,83 @@ class UtbetalingService(
             }
     }
 
+    private suspend fun getPersoner(deltakerIdenter: List<NorskIdent>): Map<PdlIdent, Pair<HentPersonBolkResponse.Person, GeografiskTilknytningResponse?>> {
+        val identer = deltakerIdenter
+            .map { ident -> PdlIdent(ident.value) }
+            .toNonEmptySetOrNull()
+            ?: return mapOf()
+
+        return pdl.hentPersonOgGeografiskTilknytningBolk(identer, AccessType.M2M).getOrElse {
+            throw StatusException(
+                status = HttpStatusCode.InternalServerError,
+                detail = "Klarte ikke hente informasjon om deltakere for kostnadsfordeling",
+            )
+        }
+    }
+
+    private suspend fun hentEnhetForGeografiskTilknytning(geografiskTilknytningResponse: GeografiskTilknytningResponse?): NavEnhetDbo? {
```

Comment:
Det er veldig sjelden nødvendig å kalle en metode med `null` som eneste input (`geografiskTilknytningResponse` er nullable i metodedeklarasjonen her).

I stedet burde metoden kreve at `geografiskTilknytningResponse` er definert (ikke nullable), og så kaller vi kun på `hentEnhetForGeografiskTilknytning` når den finnes i stedet, f.eks. `geografiskTilknytning?.let { hentEnhetForGeografiskTilknytning(it) }`